docs/helm-broker/03-05-bundles-validation.md

```diff
@@ -0,0 +1,33 @@
+a---
+title: Bundles validation
+type: Details
+---
+
+The Helm Broker validates the bundles which fetches. If some bundle does not meet the requirements, the Helm Broker won't expose it as a Service Class and it will put an information about this in the logs.
+
+To create a correct bundle follow the following rules:
+
+1. The bundle **must** contains a `meta.yaml` file with non-empty values in the following fields:
```

Comment:
I need to say that I really do not like that documentation.

We already have those sections:
https://kyma-project.io/docs/components/helm-broker#details-create-a-bundle-meta-yaml-file

there we have a table which has the column if the given filed is `required`.

by duplicating that we only generating new problem with making that always consistent. 

If that documentation is really required then imo we should add here only information about the checker script. E.g. 
- said that checker is validating if required fields are set + link to docs
- said that checker is executing the `helm lint` command and document from which helm version. (add then in Gopkg.toml add a comment above the helm dependency, that if you changing the version then change the value also in documentation).
- **maybe** add information how to execute checker locally against your bundle,
- **maybe** add information how to get logs from helm-broker in case that your bundle is not visible in Service Catalog


/cc @PK85

Reply:
I agree with you, that documentation was not needed I changed it as you proposed and now it makes more sense. Also added a section with troubleshooting and how to execute checker locally.